_posts/2018-09-06-how-we-use-pivotal-tracker-at-ombu-labs.markdown

```diff
@@ -2,155 +2,121 @@
 
 <!--more-->
 
-## Step 1: Filing user stories
+## Step 1: Writing user stories
 
-Every single thing has to be tracked according to the type of story. Most of the user
-stories are either *features* or *bugs*, although sometimes there will be *chores* to be
-done. If a *chore* reoccurs many times, then it's probably time to write a *feature*
-to reduce the time spent completing that *chore*.
+All tasks related to a project should be tracked through user stories. A good to way to think about user stories is that it is a high-level requirement card. A user story can be used to represent a technical requirement, a use case, a usage requirement or even chores the development team needs to complete. [Pivotal Tracker](https://www.pivotaltracker.com) gives you the ability to create 4 different types of stories:
 
-## Step 2: Describing user stories
+1. Feature
+2. Bug
+3. Chore
+4. Release
 
-Every [user story](https://www.mountaingoatsoftware.com/agile/user-stories) should
-be a simple description of a feature:
+Most stories will be either *features* or *bugs*, although sometimes there will be *chores* to be
+done. If a *chore* reoccurs many times, then it's probably time to write a *feature* to reduce the time spent completing that *chore*.
 
-```
-As a < type of user >, I want < some goal > so that < some business goal >.
-```
+Both project stakeholders and the Scrum team can request stories. If a developer sees a need for a story, they should create it. If the product owner needs to define a feature that should be added, they should create a story. However, new stories will always be added to the Icebox. Once the sprint is planned, stories being worked on will be dragged into the current iteration section of the project.
 
-The description of the user story should be very clear about the expected outcome.
-This will explain what is the exit criteria for the story. It is very helpful
-for designers and developers to understand what needs to happen for a user to
-be happy with the feature.
+In order to indicate priority, we organize the Icebox from most important to least important stories, from top to bottom. This way, everyone involved in the project has a clear picture of the priorities.
 
-This is an example from one of our recent projects ([https://audit.fastruby.io/](https://audit.fastruby.io/)):
+## Step 2: Describing user stories
+
+Every [user story](https://www.agilealliance.org/glossary/user-stories/) should be a description of a requirement and consist of a summary and a description.
+
+The summary is a concise description of the requirement and it clearly states what needs to be done and why. For a user story depicting a use case, for example, the following structure should be followed.
```

Comment:
```suggestion
The summary is a concise description of the requirement and it clearly states what needs to be done and why. For a user story depicting a use case, for example, we follow this structure.
```

Saying `should be followed` gives the impression that you MUST use that structure when creating stories on Pivotal Tracker.

Reply:
@arielj that's the idea. Whenever we are using user stories to represent features or use cases, we should follow that structure (we don't always do it, but we should).

Reply:
Yes, what I'm saying is that, when some external user reads that, it may look that's how _pivotal tracker_ excepts stories to be written instead of how _we_ expect stories to be written.

Reply:
Maybe it just me though, the title of the article already establishes this is how we use PT.

Reply:
I re-worded that a little but I think the structure of that section is clear enough. What do you think?

Reply:
:+1:!!